docs/network-requirements.md

```diff
@@ -0,0 +1,16 @@
+# Network Requirements
+
+Antrea has a few network requirements to get started, ensure that your hosts and firewalls allow the necessary traffic based on your configuration.
+
+| Configuration              | Host(s)                          | ports/protocols   |
+| -------------------------- | -------------------------------- | ----------------- |
+| Antrea with VXLAN enabled  | All                              | UDP 4789          |
+| Antrea with Geneve enabled | All                              | UDP 6081          |
+| Antrea with STT enabled    | All                              | TCP/UDP 1607      |
```

Comment:
Stateless Transport Tunneling (STT)
TCP 7471: Unofficial
Not sure which port is used in Antrea. @tnqn @jianjuns

Reply:
according to openvswith doc, it's using 7471 by default https://www.openvswitch.org/support/dist-docs/ovs-vswitchd.conf.db.5.html